packages/fingerprint-auth/index.android.ts

```diff
@@ -1,23 +1,42 @@
-import { Application, AndroidActivityResultEventData, Utils, AndroidApplication } from "@nativescript/core";
+import { Application, Utils } from "@nativescript/core";
 import { BiometricIDAvailableResult, ERROR_CODES, FingerprintAuthApi, VerifyFingerprintOptions, VerifyFingerprintWithCustomFallbackOptions } from "./common";
 
 declare const com: any;
 
 const KEY_NAME = "fingerprintauth";
 const SECRET_BYTE_ARRAY = Array.create("byte", 16);
-const REQUEST_CODE_CONFIRM_DEVICE_CREDENTIALS = 788; // arbitrary
+const AuthenticationCallback = (<any>androidx).biometric.BiometricPrompt.AuthenticationCallback.extend({
```

Comment:
@triniwiz Should these `androidx` calls be guarded or placed inside `lazy`? The current plugin does not contain direct androidx api calls, so I imagine this is considered breaking - we could do a new major version with these changes and only support androidx going forward?

Reply:
This look good , the lazy is more for caching things like enums as for the guarding androidx it's fine since they are using the new [lib](https://developer.android.com/jetpack/androidx/releases/biometric). However It would be nice to generate the typings to make future contributions easier for others .. 😄

Reply:
I agree this change should be a major release